etcdserver/server.go

```diff
@@ -602,7 +602,45 @@ func (s *EtcdServer) LeaderStats() []byte {
 
 func (s *EtcdServer) StoreStats() []byte { return s.store.JsonStats() }
 
+func (s *EtcdServer) isReadyToAddNewMember() bool {
+	if !s.cfg.StrictReconfigCheck {
+		// the option -strict-reconfig-check isn't passed to etcd,
+		// allow adding a new member unconditionally
+		return true
+	}
+
+	nmembers := 1
+	nstarted := 0
+
+	for _, member := range s.cluster.members {
+		if member.IsStarted() {
+			nstarted++
+		}
+		nmembers++
+	}
+
+	if nstarted == 1 {
+		// a case of adding a new node to 1-member cluster for restoring cluster data
+		// https://github.com/coreos/etcd/blob/master/Documentation/admin_guide.md#restoring-the-cluster
+
+		plog.Infof("a number of started nodes is 1, this cluster is ready to add a new node")
```

Comment:
let's move this to debug level. Also `The number of started member is 1. This cluster can accept add member request.`